tests/test_inputs.py

```diff
@@ -69,6 +70,56 @@ def module_has_entry_point(module: ModuleType):
     return any(hasattr(module, attr) for attr in ["Test", "TestStub"])
 
 
+def list_replace_nans(l: List) -> List[Any]:
+    """Replace float("nan") in a list with the string "NaN"
+
+    Parameters
+    ----------
+    l : List
```

Comment:
```suggestion
    l: List
```

Reply:
This is in the NumPy docstring spec: https://numpydoc.readthedocs.io/en/latest/format.html#sections

Reply:
It's not how the other functions are documented